static/app/components/structuredEventData/index.tsx

```diff
@@ -185,7 +185,7 @@ export function StructuredData({
     );
   }
 
-  if (typeof value === 'string') {
+  if (typeof value === 'string' || config?.isString?.(value)) {
```

Comment:
It looks like we make this check on the line below already?

Reply:
ahh you're right, i'll remove